meta-mender-variscite/recipes-kernel/kernel-modules/kernel-module-imx-gpu-viv_%.bbappend

```diff
@@ -0,0 +1 @@
+KERNEL_SRC = "git://github.com/nxp-imx/linux-imx.git;protocol=https;branch=${SRCBRANCH}"
```

Comment:
see above, misses machine override

Reply:
Added in commit 3b3279bf8ac343d3ac136a34a170dfddfc576269